static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -70,9 +70,13 @@ export function breadcrumbFactory(
     type: BreadcrumbType.ERROR,
     level: BreadcrumbLevelType.ERROR,
     category: 'exception',
-    message: error['error.value'],
+    message: error['error.value'].join(''),
     data: {
-      label: error['error.type'],
+      label: error['error.type'].join(''),
+      eventId: error.id,
+      groupId: error['issue.id'],
+      groupShortId: error.issue,
+      project: error['project.name'],
```

Comment:
Shoved all these in here so i didn't have to widen the type. It's a cheat for sure.